### ISSUE TITLE
3 Characters Needed before Searching Works

### ISSUE THREAD

**Issue by guyulmaz** [sharpshark28/my_spells#23]

**Comment by hergaiety** [comment]
@guyulmaz not able to reproduce this on the web or android versions. Please include more details on what and where the problem is so that I may look into it.

**Comment by guyulmaz** [comment]
[url=https://hizliresim.com/N1j4Mk][img]https://i.hizliresim.com/N1j4Mk.png[/img][/url]

Search not filter when I type chars and press enter

**Comment by guyulmaz** [comment]
I have seen filter start to work after 3 chars, so it is ok

**Comment by hergaiety** [comment]
@guyulmaz you're right, I'll look into making the user experience around that better or removing the minimum character limit altogether.

**Comment by hergaiety** [comment]
`json-query-chain` dependency no longer has arbitrary 3 character first limitation.

Should consider if a character limit should exist within My Spells though for performance and useful search considerations. Also will not close out this ticket until the latest `json-query-chain` is used and planned to be released into a new app version.
